app/src/main/java/com/example/sabonit/Account.java

```diff
@@ -24,6 +24,8 @@
     // The account of the user that is logged in
     private static Account currentAccount;
 
+    private ConfirmedOrders confirmedOrders;
```

Comment:
it's smart the the user holds it because logically its the user carts history

Reply:
Yes great